android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -484,7 +484,7 @@ private void doMenuAction(int menuItemId) {
                                 .provideMapFragment(), FRAGMENT_TAG_REST).commit();
                 addShadowToAppBar(true);
                 if (getSupportActionBar() != null) {
-                    getSupportActionBar().setTitle(R.string.menu_map);
+                    getSupportActionBar().setTitle("");
```

Comment:
@arp95 Because the bar has hint text in it "Search..." so we don't need title bar text.